react-native-libraries.json

```
@@ -8642,5 +8642,33 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/BabylonJS/BabylonReactNative",
```

```suggestion
    "githubUrl": "https://github.com/BabylonJS/BabylonReactNative/tree/master/Modules/%40babylonjs/react-native",
```

If package is a a part of monorepo, we need to specify full path to the folder which includes `package.json` of that library.